test.sh

```diff
@@ -9,7 +9,7 @@ echo -e "Running unit tests\n"
 nix-build -A tests --no-build-output --no-out-link --show-trace
 
 echo -e "\n\nRunning integration tests\n"
-$(nix-build  -A tests.integration-tests --no-out-link --show-trace)
+nix-build  -A tests.integration-tests --no-out-link --show-trace
```

Comment:
This was actually here to *run* the integration tests. The Nix build will just build the scripts that runs the tests. We still pretty much require this.

Reply:
oooh i see, thanks for explaining